src/client/camera.cpp

```diff
@@ -727,9 +746,11 @@ void Camera::drawNametags()
 
 Nametag *Camera::addNametag(scene::ISceneNode *parent_node,
 		const std::string &nametag_text, video::SColor nametag_color,
+		bool nametag_viewthrough_walls, bool nametag_viewthrough_liquids,
```

Comment:
This would get messier the more a nametag can be customised. It might make sense to refactor all nametag stuff into a `struct`, which would then be passed around to wherever these nametag properties are required. I'm just making a note of this - this can (or probably should?) be done in a later PR. :)

Reply:
I was actually thinking about adding some more config. Guess I'll switch it over if I do that.